libs/damap/src/lib/components/dmp/repo/repo.component.html

```diff
@@ -13,17 +13,20 @@ <h2 translate>dmp.steps.repositories.heading</h2>
 </app-step-intro>
 <app-info-message>
   {{ "info.documentation.repositoryGuidance.start" | translate }}
-  <ng-container
-    *ngFor="
-      let link of 'info.documentation.repositoryGuidance.links'
-        | translate
-        | keyvalue
-    ">
-    <a class="info-link-custom" [href]="link.url" target="_blank">{{
-      link.title
-    }}</a>
-    {{ link.text }}
-  </ng-container>
+  <a
+    class="info-link-custom"
+    [href]="'info.documentation.repositoryGuidance.links.0.url' | translate"
+    target="_blank">
+    {{ "info.documentation.repositoryGuidance.links.0.title" | translate }}
+  </a>
+  {{ "info.documentation.repositoryGuidance.links.0.text" | translate }}
+  <a
+    class="info-link-custom"
+    [href]="'info.documentation.repositoryGuidance.links.1.url' | translate"
+    target="_blank">
+    {{ "info.documentation.repositoryGuidance.links.1.title" | translate }}
+  </a>
+  {{ "info.documentation.repositoryGuidance.links.1.text" | translate }}
```

Comment:
maybe I am missing something. Why do we not keep the for loop here?

Reply:
The tests were passing before with the loop IIRC. Could you check to make it work with a loop?

Please also rebase from next-mug. Seems like there is a conflict

Reply:
No issues on my side when running tests with the old code